homeassistant/components/aprs/device_tracker.py

```diff
@@ -0,0 +1,129 @@
+"""Support for APRS device tracking."""
+
+import logging
+import threading
+
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    ATTR_LATITUDE, ATTR_LONGITUDE, PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_HOST, EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import slugify
+
+REQUIREMENTS = ['aprslib==0.6.46']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ALTITUDE = 'altitude'
+ATTR_COURSE = 'course'
+ATTR_COMMENT = 'comment'
+ATTR_FROM = 'from'
+ATTR_FORMAT = 'format'
+ATTR_POS_AMBIGUITY = 'posambiguity'
+ATTR_SPEED = 'speed'
+
+CONF_CALLSIGN = 'callsign'
+CONF_CALLSIGNS = 'callsigns'
+
+DEFAULT_CALLSIGN = 'N0CALL'
+DEFAULT_HOST = 'rotate.aprs2.net'
+
+MSG_FORMATS = ['compressed', 'uncompressed', 'mic-e']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_CALLSIGNS): cv.ensure_list,
+    vol.Optional(CONF_CALLSIGN,
+                 default=DEFAULT_CALLSIGN): cv.string,
+    vol.Optional(CONF_HOST,
+                 default=DEFAULT_HOST): cv.string
+})
+
+
+def make_filter(callsigns: list) -> str:
+    """Make a server-side filter from a list of callsigns."""
+    return ' '.join('b/{0}'.format(cs.upper()) for cs in callsigns)
+
+
+def setup_scanner(hass, config, see, discovery_info=None):
+    """Set up the APRS tracker."""
+    callsigns = config.get(CONF_CALLSIGNS)
+    fltr = make_filter(callsigns)
```

Comment:
Call it "filter". ;-)

Reply:
I did originally, but then the linters told me that I was overriding Python's built-in `filter` keyword. 😕

Reply:
What about `traffic_filter` or `callsign_filter`?

Reply:
I think I'll go with `server_filter` since it filters server-side traffic.